### PR TITLE
Modified io_check script to output variable value and change ratio

### PR DESCRIPTION
**Pull request by TanZiTT**
The io_check.r script was modified. If the variables were found different in
original and modified models.  It will print out the variable values of
both models as well as the change ratios (original values/ modified
values). "output_compare.txt" shows the sample format of output.

### CODE REVIEW

File: R_scripts/output_compare.txt [hunk 1]
```diff
@@ -4,13 +4,79 @@ Accum.Pasture
...
+0 0 NA
+0 0 NA
+6533890.55989949 98013559484.5718 6.66631290023498e-05
+6533890.55989949 98013559484.5718 6.66631290023498e-05
+6533890.55989949 98013559484.5718 6.66631290023498e-05
+0 0 NA
+0 0 NA
+0 0 NA
 
 Onsite Pets Submodel Key Variables:
```
Comment (kbrannan) [inline R_scripts/output_compare.txt:49]: Is the comaprison script working for onsite?script

**Comment by kbrannan** [comment]
Check my comment on the on-site part of the comparison. Everything else looks great!

**Comment by TanZiTT** [comment]
I see.

There's no difference on key variables between original  and modified on-site model. Thus the script printed out a NA matrix.

I will fix that.

**Comment by kbrannan** [comment]
I ran the previous version of io_check (pull request 11) and got the following output for the onsite:
Onsite Pets Submodel Key Variables:
bac.onsite.NearStrmStrctFailure.to.stream.load
Accum.RAOCUT
Different Key Variables:
Accum.RAOCUT

Shouldn't we see this change in the latest version of io_check?

**Comment by TanZiTT** [comment]
hmm, yes we should. Seems there're bugs either in the older version or the latest version scripts.
Thanks for the QA, I will look into this.

**Comment by TanZiTT** [comment]
Hi Kevin,

I just updated the script to fix the output issue.

I also ran the previous version of io_check.r script but didn’t see difference in key variables in On-site output. I also checked the output compare spreadsheet file that I shared to you long ago. The key variables in on-site outputs were the same and the output of the script is consistent with the results in spreadsheet.

I think the reason we got different results may due to the input files we used are different. The script is working fine and now if there’s no differences in key variables, it won’t print NA matrix anymore.

Now the files at my end are all up to date. Since the pull request is still open, when you merge the pull, you should be able to get the latest working script.

Regards,
Tan

**Comment by kbrannan** [comment]
Great, thanks!

**Pull request merged**